docs/operator-guide/artifacts-management/nexus-image-registry.md

```diff
@@ -0,0 +1,183 @@
+    ![Nexus ArgoCD applications](../../assets/operator-guide/artifacts-management/nexus-registry-addons-deploy.png "Nexus ArgoCD applications")
+
+4. **Sign In** into **Nexus Repository Manager** using the default credentials **Username**: `admin` and **Password**: `admin123` and update the default password for better security:
+
+    ![Nexus ArgoCD applications](../../assets/operator-guide/artifacts-management/nexus-registry-change-password.png "Nexus ArgoCD applications")
+
+5. Create a Kubernetes Secret that the **nexus-operator** will use to connect to the **Nexus Repository Manager** and create all resources:
+
+  <Tabs
+    defaultValue="manifests"
+    values={[
+      {label: 'Manifests', value: 'manifests'},
+      {label: 'External Secrets Operator', value: 'externalsecret'},
+    ]}>
+    <TabItem value="manifests">
+      ```yaml
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: nexus-admin-password
+        namespace: nexus
+      type: Opaque
+      data:
+        user: YWRtaW4=     # base64-encoded value of "admin"
+        password: cGFzcw== # base64-encoded value of "pass"
+      ```
+    </TabItem>
+    <TabItem value="externalsecret">
+      ```json
+      "nexus-admin-password":
+      {
+        "user": "XXXXXXX",
+        "password": "XXXXXXX"
+      }
+      ```
+      </TabItem>
+  </Tabs>
+
+## Configuration
+
+With [Add-ons approach](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core) `nexus-operator` create the necessary roles,**Service Account** `ci.user`, blob stores, repository, scripts, cleanup policies automatically. Below is a comprehensive guide on how to create all resources manually.
+
+## Configuration Nexus Repository Manager with [Add-ons approach](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core)
+
+  Create a secret using `manifests` or with `External Secrets Operator` to enable the **edp-nexus-operator** to connect to the **Nexus Repository Manager** for provisioning:
+
+  <Tabs
+    defaultValue="manifests"
+    values={[
+      {label: 'Manifests', value: 'manifests'},
+      {label: 'External Secrets Operator', value: 'externalsecret'},
+    ]}>
+    <TabItem value="manifests">
+      ```yaml
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: ci-nexus
+        namespace: nexus
+      type: Opaque
+      stringData:
+        password: <nexus-user-password>
+      ```
+    </TabItem>
+    <TabItem value="externalsecret">
+      ```json
+      "ci-nexus":
+      {
+        "password": "XXXXXXX"
+      }
+      ```
+      </TabItem>
+  </Tabs>
+
+## Configuration Nexus Repository Manager without [Add-ons approach](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core)
+
+Without [Add-ons approach](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core) you need to manually update: helm chart `Nexus Repository Manager` value and create blob stores, repository, roles, **Service Account** `ci.user`:
```

Comment:
For what this link?
This step about configuring without add-ons, why you add link for add-ons?

Reply:
Without add-ons, it is necessary to update and create the resources listed below:

(without 'you')